rts/System/Platform/Threading.cpp

```diff
@@ -95,7 +95,7 @@ namespace Threading {
 	static void SetWantedCoreAffinityMask(const cpu_set_t* cpuSrcSet, cpu_set_t* cpuDstSet, std::uint32_t coreMask) {
 		CPU_ZERO(cpuDstSet);
 
-		const int numCPUs = std::min(CPU_COUNT(cpuSrcSet), 32);
+		const int numCPUs = std::min(CPU_COUNT(&cpusSystem), 32);
```

Comment:
This is not strictly necessary as `cpuSrcSet` is ALWAYS cpusSystem in the engine implementation (see: [SetAffinity()](https://github.com/beyond-all-reason/spring/blob/17b0b99c0e32207236b8c924def9832646584ec1/rts/System/Platform/Threading.cpp#L161)), but it will protect against future changes.

Reply:
line 105 looks like it should be aligned with a similar change to avoid future maintenance issues (which means the parmater can be removed see previous comment.)